src/lib.rs

```diff
@@ -175,14 +175,29 @@ impl Decode for CENNZnutV0 {
 }
 
 impl Validate for CENNZnutV0 {
-    fn validate(&self, module_name: &str, method_name: &str) -> Result<(), &'static str> {
-        let module = self
-            .get_module(module_name)
-            .ok_or("CENNZnut does not grant permission for module")?;
-        module
-            .get_method(method_name)
-            .map(|_| ())
-            .ok_or("CENNZnut does not grant permission for method")
+    fn validate(
```

Comment:
Could you add a doc comment for this? It's not your tech debt but while we're here 😆